.mockery.yaml

```diff
@@ -6,6 +6,11 @@ filename: "{{.InterfaceName | snakecase}}.go"
 outpkg: mocks
 dir: "test/mocks/{{.InterfaceDirRelative}}"
 packages:
+  github.com/Dynatrace/dynatrace-operator/cmd/config:
+    interfaces:
+      Provider:
+        config:
+          mockname: "Mock{{.InterfaceName}}"
```

Comment:
would it be possible to unify `mockname` ? I see so far we prefer `Mock{{InterfaceName}}` so maybe it could be the default value for `mockname` ?

Reply:
Yeah after migrating, we can do a round of refactoring, I just reuse the same name we used before.

On another the side: package name already is: `mocks.InterfaceName` so it's better to refactor the code and use default style to not dup mock 2 times like in current implementation: `mocks.MockInterfaceName`.

Reply:
@aorcholski done in e1b4002d